rotkehlchen/db/dbhandler.py

```diff
@@ -209,6 +214,7 @@ def __init__(
         self.get_or_create_evm_token_lock = Semaphore()
         self.password = password
         self._connect()
+        self.resume_from_backup = resume_from_backup  # before calling _check_unfinished_upgrades
```

Comment:
What is the reason for storing this as a variable? Isn't it used just a single time?